auditlog_tests/tests.py

```diff
@@ -791,6 +800,129 @@ def test_unregister_delete(self):
         self.assertEqual(LogEntry.objects.count(), 0, msg="There are no log entries")
 
 
+class RegisterAllModels(TestCase):
+    def setUp(self):
+        self.test_auditlog = AuditlogModelRegistry()
+
+    def tearDown(self):
+        for model in self.test_auditlog.get_models():
+            self.test_auditlog.unregister(model)
+
+    def test_auditlog_register_models(self):
+        with self.assertRaises(TypeError):
```

Comment:
Let's use [assertRaisesMessage](https://docs.djangoproject.com/en/4.0/topics/testing/tools/#django.test.SimpleTestCase.assertRaisesMessage) to check the error message as well